.travis.yml

```diff
@@ -27,5 +27,10 @@ install:
 script:
   - ./scripts/ci_build.sh
 
-after_success:
-  - .travis/upload.sh
\ No newline at end of file
+deploy:
+  skip_cleanup: true
+  provider: script
+  script: mix hex.publish --yes
+  on:
+    tags: true
+    elixir: '1.13.4'
```

Comment:
Does the elixir need to be restated, since it's mentioned above?

Reply:
It looks like that's to limit the deploy to a specific elixir version out of the list: https://docs.travis-ci.com/user/deployment/#conditional-releases-with-on

I suppose it'd run on both if that was removed, although not sure what'd that do to hex, in terms of if one would get an error, or if it'd let it through.

Reply:
Oh, ignore me, I thought this one had a single elixir in use.